### PR TITLE
[Qt5][C++] Fixed QByteArray::toUtf8 with FILE as request body. Included <stdexcept>.

### PR DESCRIPTION
**Pull request by igor-sirotin**
* Added '#include <stdexcept>' where std::runtime_error is used
I get a build error with MSVC 16.6.30114.105 without this include:
```
error C2039: 'runtime_error': is not a member of 'std'
error C3861: 'runtime_error': identifier not found
```

* Fixed `QByteArray::toUtf8` with FILE as request body
In `api-body.mustache` there's `.toUtf8()` call on `QString output` to get a valid `QByteArray` for the following `input.request_body.append(const QByteArray&)`. This one is not needed when `output` is obtained already as `QByteArray`.

* Updated related samples
* Updated generator docs

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `5.1.x`, `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

ping @ravinikam (2017/07) @stkrwork (2017/07) @etherealjoy (2018/02) @martindelille (2018/03) @muttleyxd (2019/08)

### CODE REVIEW

File: docs/generators/fsharp-functions.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
-|sourceFolder|source folder for generated code| |OpenAPI/src|
+|sourceFolder|source folder for generated code| |OpenAPI\src|
```
Comment (MartinDelille) [inline docs/generators/fsharp-functions.md:25]: Not sure it is related to the PR.

**Review by MartinDelille — CHANGES_REQUESTED** [review]

File: docs/generators/fsharp-functions.md (hunk 1 above)
Comment (igor-sirotin) [reply docs/generators/fsharp-functions.md:25]: Well, it definitely doesn't.
But I run `./bin/utils/export_docs_generators.sh` as it's said in PR checklist, which lead to some docs changes.
Should I reverse those?

Comment (MartinDelille) [reply docs/generators/fsharp-functions.md:25]: I guess you are running the script from a windows environment. I suppose it is run on an unix environment on the CI so you should indeed revert them.

Comment (igor-sirotin) [reply docs/generators/fsharp-functions.md]: ok, done.
Thank you for your response

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by igor-sirotin** [comment]
> Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Oops, my fault!
Added this e-mail to my GitHub account. Thanks for the tip.

**Comment by etherealjoy** [comment]
@igor-sirotin 
Changes looks good to me. Were you able to compile this also on Linux?

**Comment by igor-sirotin** [comment]
No, I haven't done that.
Totally relied on CI, because the changes aren't OS-dependent.

**Review by etherealjoy — APPROVED** [review]

**Review by muttleyxd — APPROVED** [review]

**Pull request merged**